README.md

```diff
@@ -1,2 +1,173 @@
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Styles</summary>
+  <ul>
+    <li><a href="https://tailwindcss.com/">Tailwind CSS</a></li>
+  </ul>
+</details>
+
+<br>
+
+### Key Features <a name="key-features"></a>
+
+- **signup and login**
+- **create categories**
+- **create expenses and select categories for them**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<br>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Access a live demo through:
+<br>
+[this link](https://mybudget-nj8z.onrender.com/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<br>
+
+Or watch a video presentation through:
+<br>
+[this link](https://www.loom.com/share/42198050d5d44978aa9550f31b360f73)
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+- Ruby installed in your computer. ([install guide](https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/articles/ruby_installation_instructions.md))
+- Rails installed in your computer. (install guide for [Ubuntu](https://gorails.com/setup/ubuntu/21.04) and [Windows](https://gorails.com/setup/windows/10))
+- Any code editor like Vscode or Vim.
+
+### Setup
+
+Clone this repo with git. Run the following command in your terminal:
+```
+git clone https://github.com/tiagomarin/Best-Recepies.git
+
+```
+
+### Install
+
+Go to the repo directory and the following command in your terminal: 
+```
+bundle install
+
+```
+
+### Usage
+
+to see it running on your navigator run the following command in your terminal:
+```
+./bin/dev
+
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```
+  rspec
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<br>
+
+## Author <a name="author"></a>
+
+👤 **Tiago Lelinski**
+
+- GitHub: [@tiagomarin](https://github.com/tiagomarin)
+- Twitter: [@LelinskiMarin](https://twitter.com/LelinskiMarin)
+- LinkedIn: [Tiago](https://www.linkedin.com/in/tiago-lelinski-marin/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<br>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<br>
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<br>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Thanks to my learning partners for their support and advices
+
```

Comment:
- [ ] The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/). Please kindly add it 🙏

Reply:
Thanks for pointing this out too!